kolibri/core/assets/src/utils/errorReportUtils.js

```diff
@@ -6,13 +6,35 @@ class ErrorReport {
   getErrorReport() {
     throw new Error('getErrorReport() method must be implemented.');
   }
+  getDeviceInfo() {
+    return {
+      type: /Mobi|Android/i.test(navigator.userAgent) ? 'Mobile' : 'Desktop',
+      platform: navigator.platform,
```

Comment:
We can use `os` in [browserInfo.js](https://github.com/learningequality/kolibri/blob/b8aae9f6079627f49a0e1d0b89b68777aa0db8d4/kolibri/core/assets/src/utils/browserInfo.js)